src/zarr/testing/strategies.py

```diff
@@ -35,7 +35,9 @@
 paths = st.lists(node_names, min_size=1).map(lambda x: "/".join(x)) | st.just("/")
 np_arrays = npst.arrays(
     # TODO: re-enable timedeltas once they are supported
-    dtype=npst.scalar_dtypes().filter(lambda x: x.kind != "m"),
+    dtype=npst.scalar_dtypes().filter(
+        lambda x: (x.kind not in ["m", "M"]) and (x.byteorder not in [">"])
+    ),
```

Comment:
It is not clear to me how endianness works in v3. As far as I can tell, we are not handling this in a meaningful way today. 

cc @dcherian and @d-v-b

Reply:
I concur. It's not being handled today